.github/workflows/ci.yaml

```
@@ -0,0 +1,40 @@
+          node-version-file: 'package.json'
+          cache: 'pnpm'
+      - uses: actions/cache@v4
+        id: cache-npm
+        with:
+          path: node_modules
+          key: src-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
+      - run: pnpm install
+        if: steps.cache-npm.outputs.cache-hit != 'true'
+      - run: npm run build
+      - run: npm run lint
+      - run: npm run test
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)